src/Components/TaskTwo/ColourDiv.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './common.css';
+
+class DivElement extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { class: '' }
+    }
+
+    static getDerivedStateFromProps(nextProps, prevState) {
+        const className = nextProps.class
+        return { class: className }
+    }
+
+    render() {
+        return (
+            <div className={'color-div' + ' ' + this.state.class}></div>//Not sure if its right
+        )
+    }
+}
+
+export default DivElement;
```

Comment:
function ColorDiv(props){
   return <div className={'color-div' + ' ' + props.class}></div>
}
// discuss about classNames library

Reply:
class DivElement extends React.Component {
    render() {
        return (
            <div className={'color-div' + ' ' + this.props.class}></div>//Not sure if its right
        )
    }
}